content/english/secret-messages/activity-5.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Activity 5 - Ascii Code"
+date: 2022-02-23
+weight: 4
```

Comment:
All the weights need to be in order of the pages appearing on the workshop. So this should actually be weight 6.

```suggestion
weight: 6
```